sdktests/server_side_open_feature_eval.go

```diff
@@ -0,0 +1,30 @@
+package sdktests
+
+import (
+	"fmt"
+	"github.com/launchdarkly/sdk-test-harness/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/mockld"
+	"github.com/launchdarkly/sdk-test-harness/servicedef"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
+	"gopkg.in/launchdarkly/go-server-sdk-evaluation.v1/ldbuilders"
+)
+
+func doServerSideOpenFeatureEvalTests(t *ldtest.T) {
+	expectedValueV1 := ldvalue.Int(1)
```

Comment:
I've only put a test directly here to verify that things happen. This is not an example of an actual setup. I would make the common files, but I wanted to validate the other elements before doing that.